app/models/manageiq/providers/autosde/inventory/parser/storage_manager.rb

```diff
@@ -128,6 +128,8 @@ def cloud_volumes
         :name             => volume.name,
         :size             => volume.size * 1024 * 1024 * 1024,
         :ems_ref          => volume.uuid,
+        :volume_type      => "ISCSI/FC",
+        :bootable         => false,
```

Comment:
Moved these out of the persister definition since they really belong in the parser